Test/comp/compile1verbose/CompileAndThenRun.dfy.expect

```diff
@@ -16,6 +16,39 @@ hello, Dafny
 
 Dafny program verifier finished with 0 verified, 0 errors
 Wrote textual form of target program to CompileAndThenRun-java/CompileAndThenRun.java
+Additional target code written to CompileAndThenRun-java/dafny/Array.java
```

Comment:
I know this output only happens in --verbose, but it still seems more noise than needed, unfortunately. As well as being fragile when the runtime sources change. I'd prefer the previous solution of just using the runtime .class files and not recompiling them.

Reply:
> I know this output only happens in --verbose, but it still seems more noise than needed, unfortunately.

I think `/compileVerbose:1` does what it's intended to do here. We can remove that option in the test to make the test more robust, regardless of this PR, but we can also do that when it breaks.

> I'd prefer the previous solution of just using the runtime .class files and not recompiling them.

Why? Do you think there is a noticeable difference?